rpc/src/v1/helpers/errors.rs

```diff
@@ -219,4 +219,10 @@ pub fn from_transaction_error(error: EthcoreError) -> Error {
 	}
 }
 
-
+pub fn from_call_error(error: CallError) -> Error {
+	match error {
+		CallError::StatePruned => state_pruned(),
+		CallError::Execution(e) => internal("Execution error {}: ", e),
```

Comment:
This shouldn't be an internal error - it's ok if the call has failed.
I would add a separate code for this.